### ISSUE TITLE
Implement first iteration of Checkbox

### ISSUE THREAD

**Issue by dafn** [FHIDev/Fhi.Designsystem#154]
Figma: [Checkbox](https://www.figma.com/design/VgQG6XeEbFOmHIrxyAOACR/FHI-Designsystem?node-id=7277-5863&p=f&t=VmD2nxox71miL2tJ-0)

ref: https://github.com/FHIDev/Fhi.Designsystem/issues/122

acceptance criteria:
- It should be implemented in accordance with the designs in Figma
- It should be accessable from the NPM package and CDN
- It should have automated unit and accessibility tests with appropriate coverage
- It should have a story with an introduction, usage instructions, implementation examples and action interactivity
- It should conform to the Web standard on inputs/outputs.
  - e.g form interactions
- It should have appropriate component level design tokens that refrences semantic, and if appropriate, primitive level tokens.
